lib/cocoapods/installer/project_cache/project_cache_analysis_result.rb

```diff
@@ -0,0 +1,51 @@
+module Pod
+  class Installer
+    class ProjectCache
+      # The result object from analyzing the project cache.
+      #
+      class ProjectCacheAnalysisResult
```

Comment:
should this class implement `==` ?

Reply:
Don't think so? I don't use it for comparison but as a return type to grab properties from.